.github/workflows/e2e_test.yml

```diff
@@ -0,0 +1,30 @@
+name: e2e_test
+on:
```

Comment:
Consider adding that on PR, we can add the test but make it non-blocking, and tell people it is "recommended" to wait for it before merging.